mason/analyze/phenotypes.mas

```diff
@@ -0,0 +1,179 @@
+<%args>
```

Comment:
fixed in cf34b12b6106615f377247d86210534b8d078a8a (removed the file). Saved a copy of the branch in topic/analyze_phenotypes_page_bryan.